components/kyma-environment-broker/internal/process/deprovisioning/initialisation.go

```diff
@@ -24,7 +24,7 @@ const (
 
 type InitialisationStep struct {
 	operationManager  *process.DeprovisionOperationManager
-	operationStorage  storage.Provisioning
+	operationStorage  storage.Operations
```

Comment:
why did you change it?

Reply:
I will revert it. I needed it to write to the DeprovisionOperation in DB but I already deleted that after advice. Thanks!